programming_guide/section-2/section-2b/02_Broadcast/README.md

```diff
@@ -29,7 +29,7 @@ The `depth` input of an Object FIFO can also be specified as an array of integer
 
 > **NOTE:**  This functionality of the Object FIFO primitive exposes what is actually going on at the hardware level when the data movement is established for a broadcast. The object pool of the Object FIFO is not a single structure but rather composed of several pools of objects that are allocated in the memory module of each tile involved in the data movement. Specifying the `depth` as an array of integers allows the user full control to set the sizes of the pools on each individual tile.
 
-The main advantage of this feature comes to light during a situation like the one showcased in the example below, which we refer to as a broadcast with a <u>skip-connection</u>. In the example below two Object FIFOs are created: `of0` is a broadcast from producer tile A to consumer tiles B and C, while `of1` is a 1-to-1 data movement from producer tile B to consumer tile C. We refer to `of1` as a skip-connection because it is a dependency between the two consumer tiles of the same broadcast connection.
+The main advantage of this feature comes to light during a situation like the one showcased in the example below, which we refer to as a broadcast with a <u>skip-connection</u>. In the example below two Object FIFOs are created: `of0` is a broadcast from producer tile A to consumer tiles B and C, while `of1` is a 1-to-1 data movement from producer tile B to consumer tile C. We refer to `of0` as a skip-connection because since it skips over B in the A &rarr; B &rarr; C chain when connecting A &rarr; C.
```

Comment:
Thank you for adding more details @denolf ! I think either 'because' or 'since' should be removed here: ...skip-connection because / since it skips over B in the A &rarr; B &rarr; C chain when connecting A &rarr; C.